main.rb

```diff
@@ -59,31 +56,29 @@ def welcome_screen
   welcome = "*"*welcome.length + "\n" + welcome + "\n" + "*"*welcome.length
 end
 
-def setup_player(num_of_player, name, symbol)
+def setup_player(num_of_player, name)
   print "Please insert the name of the player #{num_of_player}: "
   player_name = gets.strip.capitalize
   player_name = player_name == "" ? name : player_name
-  print "Select your symbol[default: #{symbol}]: "
-  player_sym = gets.strip.upcase
-  player_sym = player_sym == "" ? symbol : player_sym
 
-  return player_name, player_sym
+  return player_name
 end
 
-def make_a_move(player, board)
+def setup_move(player, board)
   begin
```

Comment:
[The `begin` here is redundant](https://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Style/RedundantBegin)

Reply:
Deleted it